ev3/legoev3_motor.c

```diff
@@ -727,6 +729,8 @@ static enum hrtimer_restart legoev3_motor_timer_callback(struct hrtimer *timer)
 	if (ev3_tm->run_to_pos_active)
 		update_position(ev3_tm);
 
+	schedule_work(&ev3_tm->notify_state_change_work);
```

Comment:
I think so.

You might want to save your previous work in another branch in case this doesn't work out, then delete all of the other calls.